docs/rules/no-shadow.md

```diff
@@ -65,34 +65,32 @@ function foo() {
 }
 ```
 
-### `hoist`
+### hoist
 
-The option has three settings:
+The `hoist` option has three settings:
 
-* `all` - reports all shadowing before the outer variables/functions are defined.
 * `functions` (by default) - reports shadowing before the outer functions are defined.
+* `all` - reports all shadowing before the outer variables/functions are defined.
```

Comment:
Curious about this reordering. In the previous ordering, it read like a spectrum from "all" to "none" with "functions" in the middle. Can you explain the relationship with this new order?